src/dist.config.ts

```diff
@@ -42,6 +43,21 @@ function webpackConfig(args: any): webpack.Configuration {
 		return plugin;
 	});
 
+	const serviceWorker = args.pwa && args.pwa.serviceWorker;
+	if (serviceWorker) {
+		config.plugins.push(
+			new OfflinePlugin({
+				...serviceWorker,
+				ServiceWorker: {
+					entry: path.join(__dirname, 'service-worker.js'),
+					...serviceWorker.ServiceWorker,
+					minify: true
```

Comment:
According to the `offline-plugin` docs, minification is supposed to use the UglifyJsPlugin when 1) `minify` is neither `true` nor `false` and 2) the UglifyJsPlugin is already in use. However, I wasn't able to get that to work; hence the explicit `true` value.